cockatrice/src/window_main.cpp

```diff
@@ -795,3 +799,8 @@ void MainWindow::refreshShortcuts()
     aExit->setShortcuts(settingsCache->shortcuts().getShortcut("MainWindow/aExit"));
     aCheckCardUpdates->setShortcuts(settingsCache->shortcuts().getShortcut("MainWindow/aCheckCardUpdates"));
 }
+
+void MainWindow::notifyUserAboutUpdate()
+{
+    QMessageBox::information(this, tr("Information"), tr("A new version of the cockatrice client is available for download."));
```

Comment:
I somewhat like this idea, but imho just telling the user that he needs to update without telling him how could cause confusion. What about adding a servatrice settings containing a configurable url for the "latest client version download page" and adding that link to the server response, so that the clickable link appear in the QMessageBox?
The url could point by default to an url we control, like a page on our wii explaining where releases can be found. 
I can see some security implications on this.. like a malicious server being able to serve an url pointing to malware.. but if you connect to a malicious server, i'm afraid we have a lot of other scenarios where bad things can happen.